backend/search/views.py

```diff
@@ -213,6 +213,9 @@ def search_view(request):
     results, percentage, counts = \
         query.get_results(start_from, maximum_results)
 
+    if data.get('retrieveContext'):
+        results = query.augment_with_context(results, limit=settings.MAXIMUM_RESULTS)
```

Comment:
Do we need to pass a limit here? I think we want to augment with context all results that are passed to the frontend - that should not be more than 500 anyway.

Reply:
You're right. I think I first had this code run before the results were truncated.